src/routes/store/gift-cards/[slug]/+page.svelte

```diff
@@ -31,24 +31,24 @@
 
 	let count = 0, selectedDenomination = 0;
 
-	const getAmountWithoutDiscount = (/** @type {number} */ amount = 0, /** @type {number} */ commission_percent = 0) => { 
+	const getAmountWithoutDiscount = ( amount = 0, commission_percent = 0) => { 
 
-		let amount_to_pay = amount - ((amount*commission_percent) / 100);
+		let amount_to_pay = amount - ( amount * (commission_percent / 100) );
 
 		return amount_to_pay.toFixed(2);
 	};
 
-	const getAmountWithDiscount = (/** @type {number} */ amount = 0, /** @type {number} */ discount_percent = 0, /** @type {number} */ commission_percent = 0) => { 
+	const getAmountWithDiscount = ( amount = 0, discount_percent = 0, commission_percent = 0) => { 
 
-		let discounted_amount = amount - ((amount*discount_percent) / 100);
-		let amount_to_pay = discounted_amount + ((discounted_amount*commission_percent) / 100)
+		let discounted_amount = amount - ( amount * (commission_percent / 100) );
+		let amount_to_pay = discounted_amount + ( discounted_amount * (discount_percent / 100) );
 
 		return amount_to_pay;
 	 };
 </script>
 
 <svelte:head>
-  <title>PHKHotDeals | Purchase {prod_desc.name}</title>
+  <title>PHKHotDeals | Purchase {prod_desc?.name}</title>
```

Comment:
@MysticalProgrammer 

This is a one depth object chain. There is no need for the optional chaining for single depth UNLESS you are very sure that the prod_desc could be undefined.

Usually optional chaining is used when the depth is more than one.

So ```prod_desc.foo``` is very fine while ```prod_desc.foo?.baz``` is where optional chaining will come in.

I will merge this as is but note for other times.